pkg/driver/agent.go

```diff
@@ -324,8 +325,21 @@ func (ns *node) NodeGetInfo(
 	 * }
 	 */
 
-	// support all the keys that node has
-	topology := node.Labels
+	// support topologykeys from env ALLOWED_TOPOLOGIES
+	topology := map[string]string{
```

Comment:
let us not add kubernetes.io/hostname implicitely. Have this as default in the ALLOWED_TOPOLOGIES env which will add this as supported topology.

Reply:
ok, but we must keep a topology key for requrements(otherwise it will report error: no topology key found on CSINode), so I will keep "openebs.io/nodename".

Reply:
sure

Reply:
let us fix the doc also accordingly.